scripts/test_psa_compliance.py

```diff
@@ -83,7 +83,9 @@ def main(library_build_dir: str):
         # can be removed.
         if in_tf_psa_crypto_repo:
             extra_includes = ';{}/drivers/builtin/include'.format(root_dir)
-        elif os.path.isdir(os.path.join(root_dir, 'tf-psa-crypto')):
+        elif build_tree.is_mbedtls_3_6():
```

Comment:
At this point we have moved out of the mbedtls tree thus it probably does not work. Rather as we have an `in_tf_psa_crypto_repo` locla variable we should have an `is_mbedtls_3_6` one or probably even better move the  setting of `extra_includes` at the beginning of the function.

Reply:
I'm going to implement this change as you suggested, but I have a question/curiosity on this
> At this point we have moved out of the mbedtls tree thus it probably does not work

Since both https://github.com/Mbed-TLS/mbedtls/pull/9940 and https://github.com/Mbed-TLS/mbedtls/pull/9941 have a green CI, how can we tell that this change is not working?

Reply:
Good point. Actually the current directory is a descendant of the mbedtls root directory and there is a `guess_project_root` thus it was actually correct. A bit fragile probably thus the new way is quite better to me anyway.